clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -196,6 +197,44 @@ bool Sema::isKnownGoodSYCLDecl(const Decl *D) {
   return false;
 }
 
+bool isArraySizedZero(QualType Ty) {
+  if (const auto *CATy = dyn_cast<ConstantArrayType>(Ty)) {
+    const llvm::APInt size = CATy->getSize();
```

Comment:
The variable definition here seems unnecessary.

Reply:
done